thevoid/connection.cpp

```diff
@@ -19,6 +19,9 @@
 #include <boost/bind.hpp>
 #include <iostream>
 
+#include <boost/spirit/include/karma.hpp>
+#include <boost/fusion/include/std_pair.hpp>
```

Comment:
I use Boost.Spirit mainly to escape header's value that may contain nearly any characters (and I definitely don't want to code it by myself).

Related Karma example: http://boost-spirit.com/home/articles/karma-examples/generate-escaped-string-output-using-spirit-karma/comment-page-1/

Probably it's better to use Boost.Spirit to just dump header's value and all other formatting by myself.